### ISSUE TITLE
Due on 08/13: Redirects in Pension based on legacy rewriting work

### ISSUE THREAD

**Issue by mnorthuis** [department-of-veterans-affairs/va.gov-team#2212]
**User Story**
As user of VA.gov, if I click on a link or option for a legacy content page, I want to be automatically redirected to the new updated content page, so that I am always seeing the most updated, single source of truth for benefits information. 

*Issue copied from vets.gov-team repo - original issue: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/18634*

**Goal**
After all legacy rewriting work is complete, approved and live, the corresponding legacy pages are  redirected to the new versions. 

**Definition of Done:**
- [x] Determine if all parity changes are complete/live and log tickets that track the approval status
- [x] Validate previously documented redirects and update as necessary
- [x] Create a new parity redirect issue in va.gov-team repo with the list of redirects and reference the original issue and the content approval ticket
- [x] Close the old issues in vets.gov-team repo and reference the new issue in va.gov-team repo
- [x] Communicate upcoming redirects to stakeholders prior to implementing
- [x] Assign out tickets to front-end or devops as appropriate
- [x] Validate redirects in prod and update documentation when complete

Legacy URL  |  Redirect to
-- | --
https://www.benefits.va.gov/pension/current_protected_pension_rate_tables.asp | /pension/protected-pension-rates/

**Comment by mnorthuis** [comment]
Issue documenting status of rewriting: 

Pension rates page: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/18514

**Comment by MickinSahni** [comment]
PW Grooming
- Needs vetting before we create the request for Mikki. This can be closed out once that clean one is created

**Comment by lunascoop** [comment]
Redirect page is not live. @RLHecht can you check if this page is in Drupal?

**Comment by RLHecht** [comment]
@lunascoop This is another page that was built in Github and not brought over to Drupal because work was paused in between contracts. Based on the ticket Mikki linked to above it seems like there's more back and forth with the SMEs that would need to be done.

**Comment by MickinSahni** [comment]
Pulling to Backlog as we're halting on these requests

**Comment by DanielleThierryUSDSVA** [comment]
This work cannot begin until the rewriting of the page is complete. Please see new issue for rewriting: [Issue #8362](https://app.zenhub.com/workspaces/dsva-5e0105febd078d9165e53a99/issues/department-of-veterans-affairs/va.gov-team/8362).

**Comment by MickinSahni** [comment]
@lunascoop @DanielleThierryUSDSVA i slotted this for sprint 27 to give time for engineers to groom. lmk if that's still too soon

**Comment by MickinSahni** [comment]
Finalizing the date when we can implement this redirect so pulling out for now

**Comment by DanielleThierryUSDSVA** [comment]
Hi @MickinSahni @mnorthuis @lunascoop This issue is no longer blocked. The rewriting is complete and the page is published, and @bethpottsVADEPO is going to let the stakeholder know to expect an email re: the redirect. Thanks!

**Comment by bethpottsVADEPO** [comment]
@lunascoop I gave the stakeholder a heads up to expect an email from you with the timing of the redirect and other info. You can go ahead and send out your normal redirect email to the following people in the Pension group:
Alfreda.Smith@va.gov
Albert.Gayle@va.gov
Brian.Dickinson@va.gov

You don't need to cc Jen Lee anymore (we should look at that--can you put here in this ticket who you've been including?). Please cc me and:
Sandy.Tadeo@va.gov
Adrianne.Hooten@va.gov
Jessica.Tangitau@va.gov

Thanks!

**Comment by lunascoop** [comment]
@mnorthuis we are preparing to redirect the URL above. I wasn't sure if this still needs to be vetted. I also wanted to make you aware so that you can update your redirect log.

**Comment by mnorthuis** [comment]
Good to go from my end, thanks!

**Comment by lunascoop** [comment]
Thanks @mnorthuis. @bethpotts I've emailed the stakeholders and CC'd you on it. @ncksllvn @kelsonic this redirect is scheduled for 08/13.

**Comment by bethpottsVADEPO** [comment]
Thanks @lunascoop! Nice to check this off the list.

**Comment by ncksllvn** [comment]
Thanks @lunascoop! 

@MickinSahni based on the due date, I went ahead and opened the pull request for this, https://github.com/department-of-veterans-affairs/vets-website/pull/13725. It technically won't be merged til next sprint though.

**Comment by ncksllvn** [comment]
This will go live today as scheduled

**Comment by lunascoop** [comment]
This redirect has gone live and confirmed as working. Closing ticket and sending final stakeholder communication. Thanks everyone!
